src/component/App/App.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import styles from './App.module.css';
+import ButtonPanel from '../buttonPanel';
+
+function App() {
+  return (
+    <div className={styles.App}>
+      <ButtonPanel />
+    </div>
```

Comment:
- [ ] [REQUIRED] it should also render the `Display` components  ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/app/appFunctionalComponent.png))
- [ ] [REQUIRED] `Display` and `ButtonPanel` components should be wrapped in [React.Fragment](https://reactjs.org/docs/fragments.html#short-syntax) short syntax ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/app/ReactFragment.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896092/)) not in a `DIV`